### ISSUE TITLE
Module is not installable

### ISSUE THREAD

**Issue by Altai-man** [FCO/SeqSplitter#1]
```
➜  ~ zef install SeqSplitter
===> Searching for: SeqSplitter
===> Testing: SeqSplitter:ver<0.0.1>
[SeqSplitter] ===SORRY!===
[SeqSplitter] Could not find SeqSplitter at line 3 in:
[SeqSplitter]     file#/home/koto/.zef/store/SeqSplitter.git/167b5bfcd99a8328c41ac56a667ba250bfe39f38
[SeqSplitter]     inst#/home/koto/.perl6
[SeqSplitter]     inst#/home/koto/.rakudobrew/versions/moar-master/install/share/perl6/site
[SeqSplitter]     inst#/home/koto/.rakudobrew/versions/moar-master/install/share/perl6/vendor
[SeqSplitter]     inst#/home/koto/.rakudobrew/versions/moar-master/install/share/perl6/core
[SeqSplitter]     ap#
[SeqSplitter]     nqp#
[SeqSplitter]     perl5#
[SeqSplitter] ===SORRY!===
[SeqSplitter] Could not find SeqSplitter at line 2 in:
[SeqSplitter]     file#/home/koto/.zef/store/SeqSplitter.git/167b5bfcd99a8328c41ac56a667ba250bfe39f38
[SeqSplitter]     inst#/home/koto/.perl6
[SeqSplitter]     inst#/home/koto/.rakudobrew/versions/moar-master/install/share/perl6/site
[SeqSplitter]     inst#/home/koto/.rakudobrew/versions/moar-master/install/share/perl6/vendor
[SeqSplitter]     inst#/home/koto/.rakudobrew/versions/moar-master/install/share/perl6/core
[SeqSplitter]     ap#
[SeqSplitter]     nqp#
[SeqSplitter]     perl5#
===> Testing [FAIL]: SeqSplitter:ver<0.0.1>
Aborting due to test failure: SeqSplitter:ver<0.0.1> (use --force-test to override
```

You need `SeqSplitter` instead of `ChainableSeq` in provides section of META6.json to resolve this particular issue.

**Comment by Util** [comment]
Problem resolved in the just-released SeqSplitter:ver<0.0.3>.
Thanks Fernando!

**Comment by FCO** [comment]
Thank you for letting me know about that... For some reason I completely missed this issue.
